### ISSUE TITLE
Вопрос по файлу для приложения формата .аіа

### ISSUE THREAD

**Issue by N-hope** [AlexGyver/GyverMatrixWiFi#9]
Можна ли где-то выложить картинки или пдф-файл в котором показано все блоки из Thunkable.

Или Автор не имеет сейчас доступ к проекту?

**Comment by GyverLibs** [comment]
"сактебл" закрыл редактор уже очень давно

**Comment by N-hope** [comment]
Не знаю как обстоят дела, но если вы имеете доступ к аккаунту, то может сохранились проекты над которыми работали в Thunkable. 
Сейчас можно тоже сделать проект с помощью блоков, а загружать свои файлы(.аіа) нельзя. Вот и спрашиваю, есть ли возможность у вас, как автора, зайти и посмотреть свои проекты на сайт

**Comment by GyverLibs** [comment]
Нет, классический редактор полностью убран с сайта тункбл, ни создать новый проект ни открыть старый нельзя, самого редактора нет, всё

**Comment by N-hope** [comment]
Грустно
Ясно, спасибо и извините что забрала ваше время
